Client/Frontend/UIConstants.swift

```diff
@@ -9,7 +9,7 @@ public struct UIConstants {
     static let AboutHomePage = URL(string: "\(WebServer.sharedInstance.base)/about/home/")!
 
     static let AppBackgroundColor = UIColor.black
-    static let SystemBlueColor = UIColor(red: 0 / 255, green: 122 / 255, blue: 255 / 255, alpha: 1)
+    static let SystemBlueColor = UIColor(rgb: 0x0297F8)
```

Comment:
Yea. The blue felt a bit too strong. I'll double check with Bryan if he's cool with the other places where this has changed.